README.md

```diff
@@ -48,7 +48,22 @@ Output team
 
 ### Generating a gatorgrade.yml file
 
-Generate team
+The generation of yml file can be used to create assignments.
+
+In the terminal within the main directory.
+
+Use command `gatorgrade generate <path> <root directory>` to gen a yml file.
+
+- User should provide precise names of folders or files.
+
+- The user should also provide correct character case as input.
+
+- All folder(s) containing files starting with `__` or `.` will be ignore
+
+- Don't name folder or file starting with dot or double underscore.
+- These files will be ignored.
+
+All sub-dirs and files will be reserved except ones starting with `.` or `__`
```

Comment:
It's not clear what "reserved" means here - could you clarify it?

Reply:
Thanks!